.github/workflows/wheels.yml

```diff
@@ -112,63 +120,47 @@ jobs:
         if: ${{ runner.os == 'Windows' && env.IS_32_BIT == 'false' }}
 
       - name: Setup macOS
-        if: matrix.buildplat[0] == 'macos-11' || matrix.buildplat[0] == 'macos-14'
+        if: matrix.buildplat[0] == 'macos-13' || matrix.buildplat[0] == 'macos-14'
```

Comment:
The macOS 11 runner image is deprecated, and I figured it couldn't hurt to upgrade to the newest free x86 image available.